pkg/config/app_config.go

```diff
@@ -306,6 +306,10 @@ type CustomCommand struct {
 	// option where the output plays in the main panel.
 	Attach bool `yaml:"attach"`
 
+	// InvokeShell indicates whether to invoke the Command on a shell or not.
+	// Example of a bash invoked command: `/bin/bash -c "{Command}".
+	InvokeShell bool `yaml:"invokeShell"`
```

Comment:
Let me know if the description looks fine.

Reply:
I would be even more concise than that and just say `shell`